### ISSUE TITLE
Latest master crashes after minimizing

### ISSUE THREAD

**Issue by jaburjak** [xournalpp/xournalpp#2607]
**Affects versions :**
 - OS: Windows
 - Which version of libgtk do you use: 3.24.23
 - Version of Xournal++: 1.1.0+dev, 58c823f0
 - Installation method: downloaded setup from Azure Pipelines

**Describe the bug**
Xournal++ on Windows crashes every time it’s minimized. It doesn’t matter if the document is empty or not, if I draw/write anything or not, it always crashes. It happens after minimizing using the minimize button in the title bar or by clicking the Xournal++ icon in the task bar.

The crash doesn’t happen when another application is brought to the foreground – if I open Xournal++ and then switch to another program (using Alt+Tab or by clicking an icon in the task bar), Xournal++ stays open.

`…\AppData\Local\xournalpp\errorlogs` is empty.

**To Reproduce**
1. Open Xournal++
2. Minimize the window
3. After a few seconds, Xournal++ crashes

**Additional context**
If I open Windows Terminal and launch Xournal++ from there, it doesn’t crash even when I minimize it. It only works from Windows Terminal (the UWP app) though, if I launch Xournal++ from the classic cmd.exe or powershell.exe, it still crashes when I minimize it.

**Comment by Febbe** [comment]
Can you share your settings.xml, I can't reproduce this neither by compiling it directly, nor installing your nightly version.

**Comment by kubajaburek** [comment]
I just got time to look into this again, and I found out the crash keeps happening even when I delete the configuration directory in `AppData\Local\xournalpp`. But I can’t reproduce it on any of my other devices either and I also discovered that even version 1.0.20 is now crashing this way (it didn’t before). So I think I must’ve broken something on my system when I was installing and uninstalling different versions/development versions of Xournal++ and this is actually not a bug.

Can I just ask – does Xournal++ store some settings elsewhere than in `AppData\Local`, or is there something else I could try to fix it?
